packages/terra-form-select/src/multiple/Frame.jsx

```diff
@@ -428,6 +433,20 @@ class Frame extends React.Component {
     }
   }
 
+  /**
+   * Handles the mouse down events.
+   * @param {event} event - The mouse down event.
+   */
+  handleMouseDown(event) {
+    // Preventing default events stops the search input from losing focus.
+    // The default variant has no search input therefore the mouse down gives the component focus.
+    event.preventDefault();
+    if (event.target.hasAttribute('data-terra-form-select-input')) {
+      this.input.focus();
+    }
+    this.openDropdown(event);
```

Comment:
Since we are calling `this.openDropdown` on mouseDown  focus to input will be set at [openDropdown()](https://github.com/cerner/terra-core/blob/f937194e0b1d2e8b64bd4cabbbc0454d1d76cabb/packages/terra-form-select/src/multiple/Frame.jsx#L314) method. Do we still need to set the focus here as well...?

Reply:
This change is done so that  when user opens it via toggle and clicks on input to focus it in desktop.Since it is already open this [line](https://github.com/cerner/terra-core/blob/f937194e0b1d2e8b64bd4cabbbc0454d1d76cabb/packages/terra-form-select/src/multiple/Frame.jsx#L287)  is executed in openDropdown